test/src/e2e_vm_tests/test_programs/should_pass/language/generic_impl_self/src/main.sw

```diff
@@ -3,119 +3,117 @@ script;
 use std::u128::*;
 
 struct Data<T> {
-  value: T
+    value: T,
```

Comment:
I ran `forc fmt` on this so that it won't change in the future. If the line numbers change, tests will break.